aries-site/src/examples/templates/wizard/WizardAnatomy.js

```diff
@@ -0,0 +1,214 @@
+import React, { useContext, useEffect, useMemo, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import { ThemeContext } from 'styled-components';
+import { Box, Diagram, Grid, Form, ResponsiveContext, Stack } from 'grommet';
+import {
+  CancellationLayer,
+  StepFooter,
+  StepHeader,
+  WizardContext,
+  WizardHeader,
+} from './components';
+import { getWidth } from './utils';
+import { Annotation } from '../../../layouts';
+import { StepOne } from './WizardValidationExample';
+import { connection } from '../../../utils/utils';
+
+const connections = [
+  connection('1a', 'wizard-title', 'vertical'),
+  connection('1b', 'previous-button'),
+  connection('1c', 'cancel-button'),
+  connection('2b', 'step-summary'),
+  connection('2a', 'step-title'),
+  connection('2c', 'step-description'),
+  connection('3', 'step-inputs'),
+  connection('4', 'step-footer'),
+  connection('4a', 'next-step'),
+];
+
+export const WizardAnatomy = ({ containerRef }) => (
+  <Stack interactiveChild="first">
+    <Grid
+      columns={['xxsmall', 'flex', 'xxsmall']}
+      rows={['36px', '60px', '48px', '24px', '60px', '60px', 'auto', 'auto']}
+      areas={[
+        ['blah', 'annotation-1a', 'blahh'],
+        ['annotation-1b', 'wizard', 'annotation-1c'],
+        ['empty-3', 'wizard', 'empty-1'],
+        ['annotation-2b', 'wizard', 'empty-1'],
+        ['annotation-2a', 'wizard', 'empty-1'],
+        ['annotation-2c', 'wizard', 'empty-1'],
+        ['annotation-3', 'wizard', 'empty-1'],
+        ['annotation-4', 'wizard', 'annotation-4a'],
+      ]}
+      align="center"
+      justify="center"
+      gap={{ column: 'xsmall' }}
+    >
+      <WizardExample gridArea="wizard" containerRef={containerRef} />
+      <Annotation
+        alignSelf="center"
+        id="1a"
+        target="1a"
+        gridArea="annotation-1a"
+      />
+      <Annotation id="1b" target="1b" gridArea="annotation-1b" />
+      <Annotation id="1c" target="1c" gridArea="annotation-1c" />
+      <Annotation id="2b" target="2b" gridArea="annotation-2b" />
+      <Annotation id="2a" target="2a" gridArea="annotation-2a" />
+      <Annotation id="2c" target="2c" gridArea="annotation-2c" />
+      <Annotation
+        id="3"
+        target="3"
+        gridArea="annotation-3"
+        margin={{ top: 'large' }}
+      />
+      <Annotation
+        id="4"
+        target="4"
+        gridArea="annotation-4"
+        margin={{ bottom: 'large' }}
```

Comment:
why the `margin` can we not alignSelf to `end` so the 4 and 4a are closer in line to the footer and next button